.github/workflows/rust.yml

```
@@ -0,0 +1,33 @@
+name: Rust
+
+on:
+  push:
+  pull_request:
+    branches: [ "main" ]
```

Oh, I see why the CI does not show up in the PR. You set it up to run only on PRs from `main` :)
```suggestion
  push:
    branches: [ "main" ]
  pull_request
```